src/pages/experiments/[id].tsx

```diff
@@ -22,13 +25,43 @@ import { useExperiment, useHandledAsyncCallback } from "~/utils/hooks";
 import { useAppStore } from "~/state/store";
 import { useSyncVariantEditor } from "~/state/sync";
 import { HeaderButtons } from "~/components/experiments/HeaderButtons/HeaderButtons";
+import Head from "next/head";
+import { appRouter } from "~/server/api/root.router";
+import { createInnerTRPCContext } from "~/server/api/trpc";
+
+export const getServerSideProps = async (context: GetServerSidePropsContext<{ id: string }>) => {
```

Comment:
Worth keeping in mind that this will slow down our pageload because now we have to pull the stats on the server before sending down the page and pulling everything else on the client, instead of pulling everything in parallel.